packages/openactive-integration-tests/test/helpers/flow-stages/flow-stage-recipes.js

```diff
@@ -0,0 +1,85 @@
+const RequestHelper = require('../request-helper');
+const { BFlowStage } = require('./b');
+const { C1FlowStage } = require('./c1');
+const { C2FlowStage } = require('./c2');
+const { FetchOpportunitiesFlowStage } = require('./fetch-opportunities');
+const { FlowStageUtils } = require('./flow-stage-utils');
+
+/**
+ * @typedef {import('../request-state').OpportunityCriteria} OpportunityCriteria
+ * @typedef {import('../logger').BaseLoggerType} BaseLoggerType
+ * @typedef {import('../../templates/c1-req').C1ReqTemplateRef} C1ReqTemplateRef
+ * @typedef {import('../../templates/c2-req').C2ReqTemplateRef} C2ReqTemplateRef
+ * @typedef {import('../../templates/b-req').BReqTemplateRef} BReqTemplateRef
+ */
+
+/**
+ * @typedef {{
+ *   c1ReqTemplateRef?: C1ReqTemplateRef | null,
+ *   c2ReqTemplateRef?: C2ReqTemplateRef | null,
+ *   bReqTemplateRef?: BReqTemplateRef | null,
+ * }} OptionalC1C2BReqTemplateRefs
+ */
+
+const FlowStageRecipes = {
+  /**
+   * Initialise Flow Stages for a simple FetchOpportunities -> C1 -> C2 -> B flow.
+   *
+   * Rather than setting custom input for each stage, the input is just fed automatically
+   * from the output of previous stages.
+   *
+   * DO NOT USE THIS FUNCTION if you want to use custom inputs for each stage (e.g.
+   * to create erroneous requests).
```

Comment:
Perhaps clarify "create erroneous requests" doesn't mean "create requests that return errors", but rather "create malformed requests"?

Reply:
@nickevansuk could it not mean both? e.g. in core/agent-broker/implemented/customer-not-included-test.js, a request is sent to B for which an error response is expected to be returned (therefore "create requests that return errors"). You could also say that the request is malformed as `customer` is supposed to be in the request.

Alternatively, if all the spaces are used up, making a B request is not a malformed request? But it will return an error